src/OpenFeature.Contrib.Providers.FeatureManagement/OpenFeature.Contrib.Providers.FeatureManagement.csproj

```
@@ -0,0 +1,25 @@
+﻿﻿<Project Sdk="Microsoft.NET.Sdk">
```

There seems to have been a zero-width space character at the beginning of the file, which was preventing the build from succeeding. Not sure this will do the trick, but applying the suggestion below may fix that.

```suggestion
<Project Sdk="Microsoft.NET.Sdk">
```